### PR TITLE
Initial project configuration

### PR DESCRIPTION
**Pull request by andrey-gazzaev**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Adding tests
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

The project is configured with `ESLint` and `Prettier`.

## What is the new behavior?

Added eslint

## Does this PR introduce a breaking change?

- [X] Yes
- [ ] No

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+@viktor-sakhno-saritasa @andrey-gazzaev
```
Comment (gevorg-gevorgyan-saritasa) [inline .github/CODEOWNERS]: 🥺

File: packages/common/src/lib/core/services/test.service.ts [hunk 2]
```diff
@@ -11,12 +11,14 @@ import { TestMapper } from './mappers/test.mapper';
 export class TestService {
   constructor(private readonly testMapper: TestMapper) {}
 
-  public getList(): Observable<Test[]> {
+  getList(): Observable<Test[]> {
```
Comment (gevorg-gevorgyan-saritasa) [inline packages/common/src/lib/core/services/test.service.ts]: I know you want to try new things, but methods without any access modifier are confusing for me. Anyways, just my opinion.

File: packages/common/src/shared/common.module.ts [hunk 3]
```diff
@@ -1,22 +1,18 @@
-import { NgModule, Type } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { NgModule, Type } from '@angular/core';
+
 import { TestComponentComponent } from './components/test-component/test-component.component';
 
-const PRIVATE_DECLARATIONS: Type<unknown>[] = [];
+const PRIVATE_DECLARATIONS: Array<Type<unknown>> = [];
```
Comment (gevorg-gevorgyan-saritasa) [inline packages/common/src/shared/common.module.ts]: I liked `[]` more 👽

File: packages/common/src/shared/components/test-component/test-component.component.ts [hunk 4]
```diff
@@ -6,7 +6,6 @@ import { Component, Input } from '@angular/core';
   styleUrls: ['./test-component.component.css'],
 })
 export class TestComponentComponent {
-
   @Input()
-  public testModelNames: readonly string[] | undefined
+  testModelNames: readonly string[] | undefined;
```
Comment (gevorg-gevorgyan-saritasa) [inline packages/common/src/shared/components/test-component/test-component.component.ts]: Same thing about properties with modifiers. Looks like `NMBL` a bit. 🌚

**Review by gevorg-gevorgyan-saritasa — APPROVED** [review]

File: packages/common/jest.config.ts [hunk 5]
```diff
@@ -1,10 +1,10 @@
-/* eslint-disable */
 export default {
   displayName: 'common',
   preset: '../../jest.preset.js',
   setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
   coverageDirectory: '../../coverage/packages/common',
   transform: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (sakhnovikt) [inline packages/common/jest.config.ts]: Maybe, it would be better, to except such files from linter?

File: packages/web/jest.config.ts [hunk 6]
```diff
@@ -1,10 +1,10 @@
-/* eslint-disable */
 export default {
   displayName: 'web',
   preset: '../../jest.preset.js',
   setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
   coverageDirectory: '../../coverage/packages/web',
   transform: {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
```
Comment (sakhnovikt) [inline packages/web/jest.config.ts]: Disabling eslint is so not aesthetic moment

File: packages/web/src/app/app.component.ts [hunk 7]
```diff
@@ -1,15 +1,19 @@
...
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'web';
   protected readonly testModelNames$: Observable<string[]>;
+
+  @Input()
+  title = 'web';
```
Comment (sakhnovikt) [inline packages/web/src/app/app.component.ts]: I suggest try to be consistent, if you want to write properties without class modifiers, so, pls, write all properties without it, or vice versa.

**Review by sakhnovikt — APPROVED** [review]
And, It would be perfect, if you will start writing good commit messages, not `Adjust code` 🤒

**Pull request merged**